frontend/src/components/Year/Year.tsx

```diff
@@ -48,33 +49,52 @@ class YearComponent extends React.Component<Props> {
   }
 
   render() {
-    const { index, schedule } = this.props;
+    const { index, schedule, isEditable } = this.props;
     const year = schedule.years[index];
-    const isExpanded = !this.props.closedYears.has(index);
+    const isExpanded = !this.props.closedYears.has(index) || !isEditable;
 
     return (
       <div style={{ width: "100%", marginBottom: 28 }}>
-        <YearTopWrapper>
+        <YearTopWrapper isEditable={isEditable}>
           <div
             onClick={this.handleExpandedChange.bind(this)}
             style={{ marginRight: 4 }}
           >
-            {isExpanded ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+            {isEditable &&
+              (isExpanded ? (
+                <KeyboardArrowUpIcon />
+              ) : (
+                <KeyboardArrowDownIcon />
+              ))}
           </div>
           <YearTop
             year={year}
             fallStatus={schedule.yearMap[year].fall.status}
             springStatus={schedule.yearMap[year].spring.status}
             summer1Status={schedule.yearMap[year].summer1.status}
             summer2Status={schedule.yearMap[year].summer2.status}
+            schedule={schedule}
+            isEditable={isEditable}
           />
         </YearTopWrapper>
-        {isExpanded && (
+        {(!isEditable || isExpanded) && (
```

Comment:
I dont think we need this here as we have `const isExpanded = !this.props.closedYears.has(index) || !isEditable`